graphql/api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/SpecificationInput.java

```diff
@@ -33,6 +36,7 @@
   List<TagInput> tags;
   String type;
   ObjectNode configuration;
+  Map<Role, List<Principal>> security;
```

Comment:
Does this work defining it as a map here when it's `security: [Role!]!` in the graphql definition?

Reply:
no it does not ... Oversight on my part after the change to the graphql schema. Incidentally this also fixes the broken IT.